stubs/django-environ/environ/__init__.pyi

```diff
@@ -0,0 +1,11 @@
+from .compat import DJANGO_POSTGRES as DJANGO_POSTGRES, PYMEMCACHE_DRIVER as PYMEMCACHE_DRIVER, REDIS_DRIVER as REDIS_DRIVER
+from .environ import *
+
+__copyright__: str
+__version__: str
+__license__: str
+__author_email__: str
+__maintainer__: str
+__maintainer_email__: str
+__url__: str
+__description__: str
```

Comment:
I think these variables can be marked as Final (but need to import `Final` from `typing`)
```suggestion
__copyright__: Final[str]
__version__: Final[str]
__license__: Final[str]
__author_email__: Final[str]
__maintainer__: Final[str]
__maintainer_email__: Final[str]
__url__: Final[str]
__description__: Final[str]
```